draft-ietf-core-href.md

```diff
@@ -1320,10 +1322,11 @@ and URI scheme names shown in {{scheme-map}}:
 
 ### Decomposing a Request CRI into a set of CoAP Options {#decompose-coap}
 
-   The steps to parse a request's options from a CRI »cri« are as
+   The steps to parse a request's options from a CRI »cri« (and from
+   the request's intended destination IP address) are as
```

Comment:
In the interest of not having to clean up after this in https://github.com/core-wg/corrclar/issues/38, can we generalize this to something like "the CoAP message's envelope" without introducing a new term?

(And the other occurrence of the same change.)

Reply:
Is "envelope" a already-defined term?

Reply:
No it's not, and if we don't find an established term, it may be that what is in the PR now is the best we can do without creating undesired dependencies.

Maybe "(and from the request's transport address)"? That should be understood as host and port in a UDP situation, but does rule out other transports with whatever they have (slipmux being my favorite example).